src/nibpkg/kmers.nim

```diff
@@ -378,3 +373,4 @@ proc spacing_kmer*(pot: pot_t; space: int): pot_t =
         k.strand = left.strand
         k.pos = left.pos
         result.seeds.add(k)
+    echo result.seeds.len
```

Comment:
Let's print more than just an integer. Otherwise, it can be difficult to revisit later to find the line of code that produced a line of output.